src/seedu/addressbook/commands/ViewAllCommand.java

```diff
@@ -29,10 +29,10 @@ public ViewAllCommand(int targetVisibleIndex) {
     public CommandResult execute() {
         try {
             final ReadOnlyPerson target = getTargetPerson();
-            if (!addressBook.containsPerson(target)) {
```

Comment:
This here is not an example of double negative. Double negative refers to having multiple negations in the conditional (e.g. !isNotFound), and does not refer to the case of using a negated condition to represent an unusual branch.

Instead, this is a guard clause, which is used here to detect an unusual condition (i.e. the target not being found in the address book). The reason for doing this is to make the 'happy path' prominent (i.e. the target being found). You can refer to the textbook chapter: https://nus-cs2103-ay1819s2.github.io/cs2103-website/se-book-adapted/chapters/codeQuality.html.